backends/kiibohd.py

```diff
@@ -244,11 +251,20 @@ def process( self, capabilities, macros, variables, gitRev, gitChanges ):
 					# TODO Add support for Analog keys
 					# TODO Add support for LED states
 					self.fill_dict['TriggerMacros'] += "0x00, 0x01, 0x{0:02X}, ".format( triggerItem )
+					triggers_count += 1
 
 			# Add list ending 0 and end of list
 			self.fill_dict['TriggerMacros'] += "0 };\n"
 		self.fill_dict['TriggerMacros'] = self.fill_dict['TriggerMacros'][ :-1 ] # Remove last newline
 
+		print( "Triggers count:", triggers_count )
+
+		# check for too small stateWordSize
+		if stateWordSize == "8" and (triggers_count > 255 or results_count > 255):
```

Comment:
Instead of just checking for 8, also check for 16. Because it might be useful to set to 32 in some odd situations.